src/components/user/SignUpForm/styles.js

```diff
@@ -0,0 +1,52 @@
+import { StyleSheet } from 'react-native';
+import {
+  redColor,
+  blackColor,
+  whiteColor
+} from '../../../constants/styleConstants';
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    flex: 1,
+    top: 200,
+    width: 80
```

Comment:
You shouldn't use fixed values on weight or height. Try to use percentages instead.

https://medium.com/react-native-training/build-responsive-react-native-views-for-any-device-and-support-orientation-change-1c8beba5bc23

Reply:
Thanks 👌